xbmc/epg/GUIEPGGridContainer.cpp

```diff
@@ -1896,6 +1896,12 @@ void CGUIEPGGridContainer::Reset()
 
   m_lastItem    = NULL;
   m_lastChannel = NULL;
+
+  m_channels = 0;
+  m_channelCursor = 0;
+  m_channelOffset = 0;
+  m_blockCursor = 0;
+  m_blockOffset = 0;
```

Comment:
Thanks forgot about that ;) .. Now I only reset the grid coordinates if the group is changed. Therfore I've create a new method `CGUIEPGGridContainer::ResetCoordinates` and call it in `CGUIWindowPVRGuide::GetViewTimelineItems` if the group was changed.